.lando.yml

```diff
@@ -1,10 +1,14 @@
 name: sous-project
-recipe: drupal8
+recipe: drupal9
 config:
   webroot: web
-  drush: 9
-  php: '7.2'
+  php: '7.3'
+services:
+  appserver:
+    run:
```

Comment:
Same as the 8.x version. I think we might want to keep the composer install out of the lando config.

Reply:
Answered here: https://github.com/fourkitchens/sous-drupal-project/pull/11#discussion_r552126119